e2e/support/cypress.js

```diff
@@ -155,3 +155,20 @@ if (isCI) {
   // Ensure that after plugin installation is after the afterEach handling the integration.
   require("cypress-terminal-report/src/installLogsCollector")(options);
 }
+
+beforeEach(function () {
+  const isCurrentTesOss =
+    this.currentTest._testConfig.unverifiedTestConfig.tags === "@OSS";
+  const testName = this.currentTest.title;
+  if (Cypress.config("isInteractive") && isCurrentTesOss) {
+    console.log(
+      "%cSkipping test because it is tagged with @OSS:",
+      "color: red;",
+    );
+    console.log(`test name:\n ${testName}`);
+    console.log(
+      "go to cypress/support/cypress.js and temporarily remove the skipOn(true) to run the test",
```

Comment:
```suggestion
      "go to e2e/support/cypress.js and temporarily remove the skipOn(true) to run the test",
```

Reply:
I would also add that it only makes sense to run such tests against the OSS jar. So make sure MB_EDITION is "oss" when doing this.